Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst

```diff
@@ -8,427 +8,512 @@
 Advanced routing configuration (for extensions)
 ===============================================
 
-While Page based routing works out of the box, routing for extensions has to be configured explicitely in your site configuration.
+..  contents::
+    :local:
 
-.. note::
-    There is currently no graphical user interface available to configure extended routing. All adjustments need to be done via
-    manually editing your web sites' site configuration :file:`config.yaml` file (located in :file:`config/sites/<yoursite>/config.yaml`).
 
-To map `$_GET` parameters to routes, a concept called `Enhancers and Aspects` has been introduced.
+Introduction
+============
 
-An enhancer creates variants of a specific page-based route for a specific purpose (e.g. one plugin, one Extbase plugin)
-and enhances the existing route path which can then contain flexible values, so-called "placeholders".
+While page-based routing works out of the box, routing for extensions has to be
+configured explicitly in your :ref:`site configuration <sitehandling>`.
 
-On top, aspects can be registered to a specific enhancer to modify a specific placeholder, like static human readable names
-within the route path, or dynamically generated values.
+..  note::
+    There is no graphical user interface available to configure extended
+    routing. All adjustments need to be done via manual editing your website's
+    :file:`config.yaml` site configuration file (located in
+    :file:`config/sites/<yoursite>/config.yaml`).
 
-To give you an overview of what the distinction is, we take a regular page which is available at
+To map GET parameters to routes, a concept called "enhancers and aspects has
+been introduced.
+
+An :ref:`enhancer <routing-advanced-routing-configuration-enhancers>` creates
+variants of a specific page-based route for a specific purpose (e.g., one plugin,
+one :ref:`Extbase <extbase>` plugin) and enhances the existing route path, which
```

Comment:
one plugin one Extbase plugin ?  I dont get what you mean here

Reply:
A pi-based plugin, an Extbase plugin... use now just Extbase plugin.